consensus/polybft/extra.go

```diff
@@ -142,6 +142,24 @@ func (i *Extra) UnmarshalRLPWith(v *fastrlp.Value) error {
 	return nil
 }
 
+// ValidateBasic contains extra data basic set of validations
+func (i *Extra) ValidateBasic(parentExtra *Extra) error {
+	if err := i.Checkpoint.ValidateBasic(parentExtra.Checkpoint); err != nil {
```

Comment:
why do you need err check there?

Reply:
It was a leftover, good point, tnx!
Fixed https://github.com/0xPolygon/polygon-edge/pull/1160/commits/ae0ae2eb02daa81f5e7d4ad6f6ab2d68f7ee0b96